tests/contrib/starlette/app.py

```diff
@@ -0,0 +1,73 @@
+from starlette.applications import Starlette
+from starlette.responses import Response, PlainTextResponse, StreamingResponse, FileResponse
+from starlette.routing import Route
+import os
+
+app = Starlette()
+
+
+async def homepage(request):
+    response = "Success"
+    return PlainTextResponse(response)
+
+
+async def success(request):
+    response = "Success"
+    return PlainTextResponse(response)
+
+
+async def create(request):
+    response = "Created"
+    return Response(response, status_code=201, headers=None, media_type=None)
+
+
+async def error(request):
+    """
+    An example error. Switch the `debug` setting to see either tracebacks or 500 pages.
+    """
+    raise RuntimeError("Server error")
+
+
+async def server_error(request, exc):
+    """
+    Return an HTTP 500 page.
+    """
+    response = "Server error"
+    return PlainTextResponse(response)
+
+
+def stream_response():
+    yield b"streaming"
+
+
+async def stream(request):
+    return StreamingResponse(stream_response())
+
+
+async def file(request):
+    file_ptr = open("file.txt", "w+")
+    file_ptr.write("Datadog is the best!")
+    file_ptr.close()
+    return FileResponse("file.txt")
```

Comment:
We can avoid the cleanup step if we use `tempfile` and a with-context:

```
    import tempfile
    with tempfile.TemporaryFile() as fp:
        fp.write(b"Datadog says hello!")
        fp.flush()
        return FileResponse(fp.name)
```

Reply:
I actually was trying this earlier and some other stuff with tempfile but the tempfile gets removed when you return so it causes an error